arquillian-ape-sql/standalone/flyway-ftest/src/test/java/org/arquillian/ape/rdbms/flyway/PersonSchemaTest.java

```diff
@@ -0,0 +1,41 @@
+package org.arquillian.ape.rdbms.flyway;
+
+import java.net.URI;
+import org.arquillian.ape.rdbms.core.RdbmsPopulator;
+import org.assertj.db.type.Source;
+import org.assertj.db.type.Table;
+import org.h2.Driver;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.db.api.Assertions.assertThat;
+
+@RunWith(Arquillian.class)
+public class PersonSchemaTest {
+
+
+    @Flyway
+    @ArquillianResource
+    RdbmsPopulator rdbmsPopulator;
+
+
+    @Test
+    public void should_create_person_table() {
+        rdbmsPopulator.forUri(URI.create("jdbc:h2:mem:test;DB_CLOSE_DELAY=-1"))
+            .withDriver(Driver.class)
+            .withUsername("sa")
+            .withPassword("")
+            .usingDataSet("db/migration")
```

Comment:
Yeah, but this leaves user alone and forces him to jump to the javadoc of whatever we are providing support for. Maybe something to keep for the docs then?